packages/twenty-front/src/effect-components/PageChangeEffect.tsx

```diff
@@ -153,7 +153,10 @@ export const PageChangeEffect = () => {
         label: 'Create Task',
         type: CommandType.Create,
         Icon: IconCheckbox,
-        onCommandClick: () => openCreateActivity({ targetableObjects: [] }),
+        onCommandClick: () =>
+          openCreateActivity({
+            targetableObjects: [{ id: '', targetObjectNameSingular: '' }],
```

Comment:
It looks hacky, we should better handle the case where we no target inside openCreateActivity directly

Reply:
Yes agreed, let me dig in.

Reply:
I made changes in `openCreateActivityDrawer` to check if `targetTableObject` is empty, if empty no relations would be sent.